### PR TITLE
Add secondary road to proximity and model views

### PR DESCRIPTION
**Pull request by wrridgeway**
@wagnerlmichael recently calculated distances to secondary roads. This PR incorporates those distances into our proximity and modeling views.

### CODE REVIEW

File: dbt/models/proximity/columns.md [hunk 1]
```diff
@@ -355,6 +355,33 @@ Railroad locations sourced from Cook County GIS. Inclusive of any rail
 (CTA, Metra, non-passenger freight, etc.)
 {% enddocs %}
 
+## nearest_secondary_road_dist_ft
+
+{% docs column_nearest_secondary_road_dist_ft %}
+Nearest secondary road distance (feet).
+
+secondary road locations sourced from OpenStreetMap (OSM).
+secondary roads include any OSM ways tagged with `highway/secondary`
```
Comment (dfsnow) [inline dbt/models/proximity/columns.md:364]: Same for all the other docs below.

```suggestion
Secondary road locations sourced from OpenStreetMap (OSM) and
include any OSM ways tagged with `highway/secondary`
```

File: aws-athena/ctas/proximity-crosswalk_year_fill.sql [hunk 2]
```diff
@@ -38,6 +38,8 @@ WITH unfilled AS (
             AS lake_michigan_data_year,
         MAX(dist_pin_to_major_road.nearest_major_road_data_year)
             AS nearest_major_road_data_year,
+        MAX(dist_pin_to_secondary_road.nearest_secondary_road_data_year)
+            AS nearest_secondary_road_data_year,
```
Comment (dfsnow) [inline aws-athena/ctas/proximity-crosswalk_year_fill.sql:42]: **nitpick:** Technically the columns in here and the proximity table are in alphabetical order. Does it make sense to move this below, `nearest_park_data_year`?

Comment (wrridgeway) [reply aws-athena/ctas/proximity-crosswalk_year_fill.sql:42]: yep

File: aws-athena/ctas/proximity-crosswalk_year_fill.sql [hunk 3]
```diff
@@ -238,6 +247,12 @@ SELECT
             IGNORE NULLS
             OVER (ORDER BY year DESC)
     ) AS nearest_major_road_data_year,
+    COALESCE(
+        nearest_secondary_road_data_year,
+        LAST_VALUE(nearest_secondary_road_data_year)
+            IGNORE NULLS
+            OVER (ORDER BY year DESC)
+    ) AS nearest_secondary_road_data_year,
```
Comment (dfsnow) [inline aws-athena/ctas/proximity-crosswalk_year_fill.sql:255]: Still gotta move this one @wrridgeway

**Comment by wrridgeway** [comment]
this PR is looking pretty lit

**Review by dfsnow — APPROVED** [review]

**Pull request merged**